test/utilities.go

```diff
@@ -3,30 +3,297 @@ package test
 import (
 	"bufio"
 	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
 	"fmt"
+	"io/ioutil"
 	"log"
+	"os"
+	"regexp"
 	"sort"
 	"strings"
 
 	"github.com/nsf/jsondiff"
+
+	sdk_json "github.com/algorand/go-algorand-sdk/encoding/json"
+	"github.com/algorand/go-algorand-sdk/encoding/msgpack"
 )
 
+func VerifyResponse(expectedFile string, actual string) error {
```

Comment:
Moved the file loading in here so that I could process base64 files in one place, and call both equals functions.